m_tunnel_key.go

```diff
@@ -0,0 +1,144 @@
+package tc
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/mdlayher/netlink"
+)
+
+const (
+	tcaTunnelUnspec = iota
+	tcaTunnelKeyTm
+	tcaTunnelKeyParms
+	tcaTunnelKeyEncSrc
+	tcaTunnelKeyEncDst
```

Comment:
These consts have to be in sync with https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/tc_act/tc_tunnel_key.h#L25
So to merge `KeyEncSrc` and `KeyEncDst` for the IPv6 and IPv4 case only works for the Go struct `TunnelKey` but you have to differentiate when communicating with the kernel.